lib/results.go

```diff
@@ -205,9 +216,9 @@ func headerBytes(h http.Header) []byte {
 }
 
 // NewCSVDecoder returns a Decoder that decodes CSV encoded Results.
-func NewCSVDecoder(r io.Reader) Decoder {
-	dec := csv.NewReader(r)
-	dec.FieldsPerRecord = 12
+func NewCSVDecoder(rd io.Reader) Decoder {
+	dec := csv.NewReader(rd)
+	dec.FieldsPerRecord = 13
```

Comment:
This is one change I don't feel very comfortable with. If someone has a result encoded to csv saved to a file by an older version and tries to read it re-encode it, new versions will not be able to read it for decoding.

How do you feel about the options between not setting `FieldsPerRecord`, or keeping it and dealing with the fallout?